src/main/java/core/basesyntax/strategy/operations/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.operations;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
Apply to all handlers ;)